packages/shared/src/graphql/actions.ts

```diff
@@ -2,6 +2,13 @@ import request, { gql } from 'graphql-request';
 import { graphqlUrl } from '../lib/config';
 
 export enum ActionType {
+  CreateSquad = 'createSquad',
```

Comment:
Just make sure these match the backend as I think we use snake_case there.

Reply:
Done